test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3462,6 +3462,48 @@ def forward(self, x):
         x = torch.randn(3, 3)
         self.run_test(MyReluModule(), x)
 
+    # Clip <= opset 11 does not support non-floats (e.g. ints)
+    # We enable such non-float types to avoid generating wrong ONNX file
+    # and enhance the compatibility of pytorch.onnx.
+    @skipIfUnsupportedMaxOpsetVersion(11)
```

Comment:
Why are these tests not valid for opset version `> 11`?

Same question goes for all the other tests below.

Reply:
I intended to only test the casting functionality. But you are right we should also test the full cases since they are not covered by previous tests. (prior tests only look at float inputs not other inputs).

Reply:
@ganler Are you going to remove the `@skipIfUnsupportedMaxOpsetVersion(11)` then?

Reply:
I removed the that for other ones. But this one (Clip) is kept to check the CI before ORT supports clip12<int>.

Reply:
Still waiting on https://github.com/microsoft/onnxruntime/pull/10701. Will ping ORT team again